packages/pages-shared/environment-polyfills/miniflare.ts

```diff
@@ -1,11 +1,13 @@
 import { polyfill } from ".";
 
 export default async () => {
+	const { HTMLRewriter } = await import("@miniflare/html-rewriter");
```

Comment:
Hmm. I don't know if we want to start depending upon the "old" Miniflare here.
Why does this need to be polyfilled?
Isn't it available inside workerd already?

Reply:
Well I can see that @mrbbot approved so perhaps this is fine.